### PR TITLE
Refactor addArea page

### LINKED ISSUE #139: [IMPROVEMENT]: AddArea page more beautiful
### Description

Make the AddArea page more beautiful.
Manage action and reaction display better.
Show action at first.
On action select, show reaction.
On reaction select, show resume, with area name creation.
Create button

### To do

Create a design on figma
Apply the design on web page

### Priority

Medium

### Project parts

- [ ] Server
- [ ] Application
- [X] Web
- [ ] Automation

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by osvegn**
# Updated feature
The area page have been fully changed

image
image
image

### CODE REVIEW

File: Web/src/AddAreaPage.js [hunk 1]
```diff
@@ -5,24 +5,34 @@ import LogoTwitch from './assets/twitch.png';
...
 
 /**
  * @brief Return the AddArea page for AREA
  * This page will be updated soon
  */
 export default function AddAreaPage(props) {
-    const [selectedActionIndex, setSelectedActionIndex] = useState(0)
-    const [selectedReactionIndex, setSelectedReactionIndex] = useState(0)
+    // const [selectedActionIndex, setSelectedActionIndex] = useState(0)
+    // const [selectedReactionIndex, setSelectedReactionIndex] = useState(0)
```
Comment (STom6) [inline Web/src/AddAreaPage.js]: variable in commentary perhaps unnecessary?

**Comment by JohanCDev** [comment]
Make the titles Capitalized as in the app with the property text-transform in the css that is linked

**Review by JohanCDev — DISMISSED** [review]
Lgtm

File: Web/src/AddAreaPage.js [hunk 2]
```diff
@@ -43,124 +51,265 @@ export default function AddAreaPage(props) {
...
+            index: index,
+        })
+    }
+    function goSelectAction() {
+        updatePageIndex(0)
+    }
+    function goSelectReaction() {
+        updatePageIndex(1)
+    }
+    function goResume() {
```
Comment (STom6) [inline Web/src/AddAreaPage.js:98]: doc

File: Web/src/AddAreaPage.js [hunk 3]
```diff
@@ -19,10 +20,17 @@ import uuid from 'react-native-uuid';
  * This page will be updated soon
  */
 export default function AddAreaPage(props) {
-    const [selectedActionIndex, setSelectedActionIndex] = useState(0)
-    const [selectedReactionIndex, setSelectedReactionIndex] = useState(0)
     const navigate = useNavigate();
 
+    const [pageInfo, setPageInfo] = useState({
+        title: ["Selectionne une action", "Selectionne une réaction", "Choisie le titre de ton area"],
```
Comment (JohanCDev) [inline Web/src/AddAreaPage.js]: Faute d'orthographe ("Choisie" -> "Choisis")

**Review by JohanCDev — CHANGES_REQUESTED** [review]

**Review by STom6 — DISMISSED** [review]
lgtm

**Review by 31Nathan — DISMISSED** [review]
LGTM

**Comment by JohanCDev** [comment]
You should see with @STom6 how to change the Areas.ts file, it's missing some critical informations for it to work

**Review by 31Nathan — APPROVED** [review]
LGTM

**Review by JohanCDev — APPROVED** [review]
lgtm

**Pull request merged**